pkg/controller/volume/persistentvolume/scheduler_binder.go

```diff
@@ -300,10 +345,8 @@ func (b *volumeBinder) BindPodVolumes(assumedPod *v1.Pod) (err error) {
 	}
 
 	return wait.Poll(time.Second, b.bindTimeout, func() (bool, error) {
-		// Get cached values every time in case the pod gets deleted
-		bindings = b.podBindingCache.GetBindings(assumedPod, assumedPod.Spec.NodeName)
-		claimsToProvision = b.podBindingCache.GetProvisionedPVCs(assumedPod, assumedPod.Spec.NodeName)
-		return b.checkBindings(assumedPod, bindings, claimsToProvision)
+		b, err := b.checkBindings(assumedPod, bindings, claimsToProvision)
```

Comment:
Is the pod deletion case still handled?

Reply:
Oh, I missed this, binding operation should be cancelled when pod is deleted or assigned a node by users or other scheduler.

Done in 7140fda91d1a239862b3347977fafe27efab1994.